remote-processor/RemoteCommandHandlerTemplate.h

```diff
@@ -112,144 +137,95 @@ class TRemoteCommandHandlerTemplate : public IRemoteCommandHandler
         }
 
     private:
-        std::string _strCommandName;
-        RemoteCommandParser _pfnParser;
-        uint32_t _uiMinArgumentCount;
-        std::string _strHelp;
-        std::string _strDescription;
-    };
-
-public:
-    TRemoteCommandHandlerTemplate(CCommandParser* pCommandParser) : _pCommandParser(pCommandParser), _uiMaxCommandUsageLength(0)
-    {
-        // Help Command
-        addCommandParser("help", NULL, 0, "", "Show commands description and usage");
-    }
-    ~TRemoteCommandHandlerTemplate()
-    {
-        uint32_t uiIndex;
 
-        for (uiIndex = 0; uiIndex < _remoteCommandParserVector.size(); uiIndex++) {
+        /** Command Handler callback */
+        RemoteCommandParser mParser;
 
-            delete _remoteCommandParserVector[uiIndex];
-        }
-    }
+        /** Needed arguments number */
+        uint32_t mMinArgumentCount;
 
-    // Parsers
-    bool addCommandParser(const std::string& strCommandName,
-        RemoteCommandParser pfnParser,
-        uint32_t uiMinArgumentCount,
-        const std::string& strHelp,
-        const std::string& strDescription)
-    {
-        if (findCommandParserItem(strCommandName)) {
+        /** Command help string */
+        std::string mHelp;
```

Comment:
"help" isn't clear enough about what it is. What it actually contains is the description of the arguments.